core/src/main/scala/com/databricks/labs/remorph/parsers/snowflake/SnowflakeAstBuilder.scala

```diff
@@ -90,9 +90,19 @@ class SnowflakeAstBuilder(override val vc: SnowflakeVisitorCoordinator)
     errorCheck(ctx) match {
       case Some(errorResult) => errorResult
       case None =>
-        val ctes = vc.relationBuilder.visitMany(ctx.commonTableExpression())
-        ir.WithCTE(ctes, relation)
+        if (ctx.RECURSIVE() == null) {
+          val ctes = vc.relationBuilder.visitMany(ctx.commonTableExpression())
+          ir.WithCTE(ctes, relation)
+        } else {
+          ir.UnresolvedCommand(
+            ruleText = contextText(ctx),
+            message = "*** WITH RECURSIVE IS NOT SUPPORTED ***",
```

Comment:
there are reports that `WITH RECURSIVE` could be implemented with various slow-performance hacks in PySpark - see https://stackoverflow.com/questions/52562607/recursive-cte-in-spark-sql

given that, we should emit new `WithRecursive` node and try to translate it in PySpark later

Reply:
Sure

Reply:
Ready for review